hkmc2/shared/src/test/mlscript-compile/Runtime.mls

```diff
@@ -29,7 +29,7 @@ class EffectHandle(_reified) with
   fun resumeWith(value) =
     Runtime.try(() => resume(reified.contTrace)(value))
   fun raise() =
-    topLevelEffect(reified)
+    topLevelEffect(reified, null)
```

Comment:
I added a `null` argument here because `topLevelEffect` takes 2 arguments.

Reply:
Smart. `null` is the best value.